readthedocs/proxito/urls.py

```diff
@@ -108,6 +108,19 @@
         name='docs_detail',
     ),
 
+    # Hack /en/latest so it redirects properly
+    # We don't want to serve the docs here,
+    # because it's at a different level of serving so relative links break.
```

Comment:
Why we don't make the previous URLConf with optional trailing slash instead of creating a new one?

Reply:
That is what the comment is explaining :)